asteroid/utils.py

```diff
@@ -236,6 +236,22 @@ def flatten_dict(d, parent_key='', sep='_'):
     return dict(items)
 
 
+def sanitize_configs(conf_dict):
```

Comment:
Just to be safe, move this to `System` and modify only the config that's going to be logged to tensorboard (`hparams`)